src/providers/FileSystemProvider/FileSystemProvider.ts

```diff
@@ -47,40 +48,15 @@ export class FileSystemProvider implements vscode.FileSystemProvider {
     if (!api.active) {
       return;
     }
-    const sql = `CALL %Library.RoutineMgr_StudioOpenDialog(?,?,?,?,?,?,?)`;
     const { query } = url.parse(uri.toString(true), true);
-    const type = query.type && query.type != "" ? query.type.toString() : "all";
     const csp = query.csp === "" || query.csp === "1";
-    let filter = "";
-    if (query.filter && query.filter.length) {
```

Comment:
It still functions correctly in my test environment.  The new utility class handles processing the filter queryparam.  Since the call to the actionQuery was moved out of this and into the helper, the support for the filter queryparam moved as well.

Reply:
OK, I can now see where this happens :+1: